exporters/common/src/main/java/io/opentelemetry/exporter/internal/http/HttpExporterBuilder.java

```diff
@@ -124,6 +135,16 @@ public HttpExporterBuilder<T> setMeterProvider(Supplier<MeterProvider> meterProv
     return this;
   }
 
+  public HttpExporterBuilder<T> setHealthMetricLevel(HealthMetricLevel healthMetricLevel) {
```

Comment:
I believe you need to include health metric level and the component type in the toString implementation. Same applies to `GrpcExporterBuilder`.